platform/barefoot/bfn-modules/debian/control

```diff
@@ -7,6 +7,5 @@ Standards-Version: 3.9.3
 
 Package: bfn-modules
 Architecture: amd64
-Depends: linux-image-4.9.0-11-2-amd64
```

Comment:
>linux-image-4.9.0-11-2-amd64 [](start = 9, length = 28)

It should depend on another version of linux-image?

Reply:
I just remove this line so that we do not need to change when we upgrade the kernel.

Reply:
I know it is convenient. But is it the right thing to do? 

---
In reply to: [406544761](https://github.com/Azure/sonic-buildimage/pull/4405#discussion_r406544761) [](ancestors = 406544761)

Reply:
all the modules are built in the context of sonic buildimage, therefore we do not have such issue to install the packages for other kernel.

Reply:
i restored those depends.